### PR TITLE
Travisify

### PR DESCRIPTION
**Pull request by Manishearth**

### CODE REVIEW

**Comment by bjz** [comment]
This is failing because we cfg the modules for osx only: https://github.com/servo/rust-cocoa/blob/master/src/lib.rs#L20-L23

**Comment by Ms2ger** [comment]
@Manishearth should we try to get osx capacity, or just move this to buildbot?

**Comment by Manishearth** [comment]
Either is fine by me

**Comment by Ms2ger** [comment]
@frewsxcv thoughts?

**Comment by frewsxcv** [comment]
Let me reply back to Travis CI and see if I can get this repository whitelisted for OSX support (along with a few others)

**Comment by frewsxcv** [comment]
Reopened to rerun the tests

**Comment by frewsxcv** [comment]
Travis CI replied and informed me that they enabled OSX support for the other projects:

* https://github.com/servo/cocoa-rs
* https://github.com/servo/cgl-rs
* https://github.com/servo/io-surface-rs
* https://github.com/servo/rust-layers

I'll open PRs for them

**Comment by frewsxcv** [comment]
Tests pass; this should be good to r+

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+os: osx
+
+language: rust
+sudo: false
+
+script:
```
Comment (frewsxcv) [inline .travis.yml]: This section is unnecessary since it does `build` and `test` by default

**Pull request merged**